docs/deploy/replication.mdx

```diff
@@ -9,7 +9,11 @@ title: pg_search
   for access.
 </Note>
 
-In this guide, we'll walk through setting up streaming replication for `pg_search`. This setup allows you to replicate data from a primary server to one or more standby servers.
+In production settings, we recommend deploying ParadeDB as a replica of your primary Postgres. This allows ParadeDB to
```

Comment:
@rebasedming Do you think we want to add a sentence to explain to people they can use kubernetes or docker here?

".. as a replica of your primary Postgres. You can do so via Kubernetes (link our helm-chart or somewhere else in our docs to deploy kubernets) or Docker (similar link)."

Just a suggestion